.buildkite/commands/release-build-wordpress-internal.sh

```diff
@@ -20,4 +20,6 @@ echo "--- :closed_lock_with_key: Installing Secrets"
 bundle exec fastlane run configure_apply
 
 echo "--- :hammer_and_wrench: Building"
-bundle exec fastlane build_and_upload_app_center skip_confirm:true
+bundle exec fastlane build_and_upload_app_center
```

Comment:
Here's your mistake: you forgot the `\` at the end of the line so that the lines below would be considered the continuation of this one. That's why the CI failed, because the `skip_confirm:true` were not actually passed to the lane due to that missing `\` here.
```suggestion
bundle exec fastlane build_and_upload_app_center \
```